penny_university_frontend/src/index.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import {BrowserRouter as Router} from 'react-router-dom'
-import {applyMiddleware, createStore} from 'redux'
+import {applyMiddleware, createStore, compose} from 'redux'
 import thunk from 'redux-thunk'
 import rootReducer from './reducers'
 import api from './middleware/api'
 import './index.css'
 import './style.css'
 import App from './App'
 
-const store = createStore(
-  rootReducer,
+// Eventually we will want to move this into a DEV configuration
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const store = createStore(rootReducer, composeEnhancers(
```

Comment:
what's an enhancer?

Reply:
In this context, something we are connecting to that doesn't really exist within the application.

Reply:
Anything that "enhances" our store, outside of what is provided by redux.